src/CommonLib/aspnetcoreconfig.h

```diff
@@ -168,11 +169,17 @@ class ASPNETCORE_CONFIG : IHttpStoredContext
     }
 
     BOOL
-        QueryStdoutLogEnabled()
+    QueryStdoutLogEnabled()
     {
         return m_fStdoutLogEnabled;
     }
 
+    BOOL
+    QueryWebSocketEnable()
```

Comment:
QueryWebSocketEnable**d** to keep name consistency?

Reply:
good point